.github/workflows/main.yml

```diff
@@ -27,6 +27,10 @@ jobs:
       env:
         COMPOSER_ROOT_VERSION: dev-master
 
+    - name: Run Xulieta
+      if: matrix.dependencies == 'authoritative'
+      run: ./bin/xulieta check:erromeu tests/assets/syntax-error-checkstyle.md --output=checkstyle | ./vendor/bin/cs2pr
+
```

Comment:
**To be removed before merge! It is here only to test the GitHub Action message.**